### ISSUE TITLE
Text vs background should have contrast ratio of at least 4.5:1 

### ISSUE THREAD

**Issue by ajpeddakotla** [hypothesis/product-backlog#786]
This issue is part of the larger accessibility epic written up here #753

Problem:
We failed in this accessibility check under the Contrast section because of the contrast difference between text and our background. We need to fix this so that we can ensure that users can read text that is presented over a background. For more information on this, read the W3C note on this here: https://www.w3.org/TR/WCAG20-TECHS/G18.html

Acceptance Criteria:
- [ ] Improve contrast between text and background to minimum standards at least (4:5:1).
- [ ] We need to evaluate how this affects the customized version of the clients (i.e. is this something we should make customizable, or should we demand this ratio for every implementation of the client)?

**Comment by ajpeddakotla** [comment]
Scored as 5 based on LOE, Community, User engagement.

**Comment by ajpeddakotla** [comment]
Closed in favor of #1085
